tools/test-proxy/Azure.Sdk.Tools.TestProxy/Matchers/CustomDefaultMatcher.cs

```diff
@@ -18,21 +18,23 @@ public class CustomDefaultMatcher : RecordMatcher
         /// <param name="excludedHeaders">A comma separated list of additional headers that should be excluded during matching.</param>
         /// <param name="ignoredHeaders">A comma separated list of additional headers that should be ignored during matching.</param>
         /// <param name="ignoreQueryOrdering">By default, the test-proxy does not sort query params before matching. Setting true will sort query params alphabetically before comparing URI.</param>
-        public CustomDefaultMatcher(bool compareBodies = true, string excludedHeaders = "", string ignoredHeaders = "", bool ignoreQueryOrdering = false) : base(compareBodies: compareBodies, ignoreQueryOrdering: ignoreQueryOrdering)
+        /// <param name="ignoredQueryParameters">A comma separated list of query parameterse that should be ignored during matching.</param>
+        public CustomDefaultMatcher(bool compareBodies = true, string excludedHeaders = "", string ignoredHeaders = "", bool ignoreQueryOrdering = false, string ignoredQueryParameters = "") 
+            : base(compareBodies: compareBodies, ignoreQueryOrdering: ignoreQueryOrdering)
         {
             foreach(var exclusion in excludedHeaders.Split(",").Where(x => !string.IsNullOrEmpty(x.Trim())))
             {
-                if (!ExcludeHeaders.Contains(exclusion)){
-                    ExcludeHeaders.Add(exclusion);
-                }
+                ExcludeHeaders.Add(exclusion);
             }
 
             foreach (var exclusion in ignoredHeaders.Split(",").Where(x => !string.IsNullOrEmpty(x.Trim())))
             {
-                if (!IgnoredHeaders.Contains(exclusion))
-                {
-                    IgnoredHeaders.Add(exclusion);
-                }
+                IgnoredHeaders.Add(exclusion);
+            }
+
+            foreach (var exclusion in ignoredQueryParameters.Split(",").Where(x => !string.IsNullOrEmpty(x.Trim())))
+            {
+                IgnoredQueryParameters.Add(exclusion);
```

Comment:
Could this be simplified to this?:
```suggestion
            var exclusions = ignoredQueryParameters.Split(",", StringSplitOptions.RemoveEmptyEntries | StringSplitOptions.TrimEntries);
            IgnoredQueryParameters.Add(exclusions);
```